roles/devscripts/templates/lv.j2

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 92-create-logical-volume
+  labels:
+    machineconfiguration.openshift.io/role: {{ ocp_role }}
+    service: cinder
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+{% for _disk in _disk_names %}
+        - device: "/dev/{{ _disk }}"
+          wipeTable: true
+          partitions:
+            - label: "data.disk.{{ loop.index }}"
+              number: 1
+              sizeMiB: 0
+              startMiB: 0
+{% endfor %}
+    systemd:
+      units:
+        - name: lv-cinder-volumes.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create logical volume with name cinder-volumes.
+            Conflicts=umoun.target
+            After=var.mount lvm2-monitor systemd-udev-settle.service
+
+            [Service]
+            Type=oneshot
+            ExecStart=/usr/local/sbin/lv-cinder-volumes.sh
+
+            [Install]
+            WantedBy=multi-user.target
+    files:
+      - path: /usr/local/sbin/lv-cinder-volumes.sh
+        overwrite: false
+        mode: 493
+        user:
+          name: root
+        group:
+          name: root
+        contents:
+          source: data:text/plain;charset=utf-8;base64,IyEgL3Vzci9iYXNoDQpzZXQgLWV1byBwaXBlZmFpbA0KDQppZiBbWyAkKGx2ZGlzcGxheSAvZGV2L2NpbmRlcl92b2x1bWVzX3ZnL2NpbmRlci12b2x1bWVzKSBdXTsgdGhlbg0KICAgIGVjaG8gImNpbmRlci12b2x1bWVzIHZnIGV4aXN0cy4iDQogICAgZXhpdCAwDQpmaQ0KDQpkaXNrcz0kKGxzYmxrIC1vIE5BTUUsVFlQRSB8IGF3ayAneyBpZiAoJDIgPT0gImRpc2siICYmICQxICE9ICJzZGEiKSBwcmludCAiL2Rldi8iJDF9JykNCmRpc2tfc3RyPScnDQoNCmZvciBkaXNrIGluICR7ZGlza3N9OyBkbw0KICAgIHB2Y3JlYXRlICR7ZGlza30NCiAgICBkaXNrX3N0cj0iJHtkaXNrX3N0cn0gJHtkaXNrfSINCmRvbmUNCg0KdmdjcmVhdGUgY2luZGVyX3ZvbHVtZXNfdmcgJHtkaXNrX3N0cn0NCmx2Y3JlYXRlIC1sIDEwMCVGUkVFIC1uIGNpbmRlci12b2x1bWVzIGNpbmRlcl92b2x1bWVzX3Zn
```

Comment:
What's this b64 encoded content? the same content as the `lv-cincer-volumes.sh` script exposed in the `files` directory ? If so, would it be possible to generate that b64 string from the script?